    public void test_null_newName() throws Exception
    {
        boolean exThrown = false;
        try {
            xmlobject.substgroup.OrderItem order = OrderItem.Factory.newInstance();
            order.substitute(null, OrderItem.type);
        } catch (IllegalArgumentException iaEx) {
            exThrown = true;
        }
        if (exThrown != true)
            throw new Exception("Exception was not thrown");
    }

